desktop/src/backends/navigator.rs

```diff
@@ -46,6 +60,8 @@ impl ExternalNavigatorBackend {
         proxy: Option<Url>,
         upgrade_to_https: bool,
         open_url_mode: OpenURLMode,
+        socket_allowed: HashSet<String>,
+        socket_behavior: SocketBehavior,
     ) -> Self {
```

Comment:
This new function triggers clippy::too_many_arguments lint. Should we allow it here?